scrapeNews/scrapeNews/settings.py

```diff
@@ -9,15 +9,21 @@
 #     http://scrapy.readthedocs.org/en/latest/topics/downloader-middleware.html
 #     http://scrapy.readthedocs.org/en/latest/topics/spider-middleware.html
 import logging
-
 BOT_NAME = 'scrapeNews'
 
 SPIDER_MODULES = ['scrapeNews.spiders']
 NEWSPIDER_MODULE = 'scrapeNews.spiders'
 
-# Make Changes here to edit logging behaviours
-LOG_LEVEL = 'WARNING'  # to only display errors
-LOG_FORMAT = '%(levelname)s: %(message)s'
+#Logger Configuration
+LOG_LEVEL = 'CRITICAL'  # to only display errors
+logger = logging.getLogger("scrapeNews")
+formatter = logging.Formatter('%(filename)s[%(lineno)d]: %(message)s')
```

Comment:
Check for these settings for formatter; If you object them; change and merge!

Reply:
@thisisayush

Reply:
Noted! Feel free to comment out these lines and inject your own; I would still prefer my settings; Hence i would request you not to remove them; I'll add your settings configuration (Commented at the bottom of the page before merging this! :) )